node/src/components/consensus/highway_core/finality_detector/rewards.rs

```diff
@@ -0,0 +1,363 @@
+use tracing::trace;
```

Comment:
This whole file can be removed, it's no longer necessary on feat-2.0 (or at least, shouldn't be).

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4